src/juris-fr-desc.json

```diff
@@ -1,64 +1,99 @@
 {
   "courts": {
-    "ca": {
-		"name": "Cour d'appel",
-		"abbrev": "CA >"
-    },
     "cass": {
-      "name": "Cour de Cassation"
+      "name": "Cour de cassation",
+	"abbrev": "Cass."	    
     },
-    "cass~civ1": {
-      "name": "Cour de Cassation|1re chambre civile"
+    "cass~civ.1": {
+      "name": "< 1re Chambre civile"
```

Comment:
A comma is missing here and between many other "name": "..." and "abbrev": "..."